sphinx_click/ext.py

```diff
@@ -491,8 +495,6 @@ def _generate_nodes(
         :returns: A list of nested docutil nodes
         """
         ctx = click.Context(command, info_name=name, parent=parent)
-        if nested == NESTED_COMPLETE:
-            print("Nested with COMPLETE")
```

Comment:
Can you avoid adding this in the prior commit, please

Reply:
Oh damn, I'm really sorry, I was sure I'd removed all that stuff...
Will be removed, of course.